### PR TITLE
Reorg documentation

### PR DESCRIPTION
**Pull request by jman-schief**
Hi @brotzeit 

here's a first draft of some ideas for the documentation [available at this link](https://emacs-ng.github.io/emacs-ng/).

- Moving the building instruction in a section on its own
- Mention the CI binary releases for Ubuntu (maybe move them at the top?)

I'm not sure if the requirements should be split in
- requirements to run emacs-ng (in getting-started.md)
- requirements to build emacs-ng (in building.md)

opinions? Happy to receive feedback, suggestions, ideas from everyone :)

thanks

### CODE REVIEW

File: docs/building.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+You may need to run `sudo make install` depending on your system configuration.
+
+Now emacs should be available at `./src/emacs`. We can launch the application
+via `./src/emacs`. We can navigate to the lisp scratchpad by pressing C-x b and
+hitting enter.
+
+### Using Docker
+
+If you want to build packages without installing all the tooling, you can use a Docker container. Our CI builds [releases for Ubuntu 20](https://github.com/emacs-ng/emacs-ng/releases), you can find instruction on how to build with Docker [at this link](https://github.com/emacs-ng/emacs-ng/tree/master/docker).
```
Comment (brotzeit) [inline docs/building.md:36]: Have you tried the image ?

File: docs/building.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+These are instructions to build emacs-ng on your workstation. First [check the requirements][compile-req].
+
+[compile-req]: https://emacs-ng.github.io/emacs-ng/getting-started/index.html#requirements
```
Comment (brotzeit) [inline docs/building.md:3]: Maybe we should pull the requirements and everything else that is related to building emacs-ng ?

**Comment by brotzeit** [comment]
What do you think about something like
```
  - Home:
    - Overview: README.md
    - Setup:
      - Build from source: building.md
      - Release deb package: release.yml
      - Nix Build / Develop: nix-develop.md
    - Getting started: getting-started.md
```

Afterwards, I would put the javascript related stuff together. But you don't have to do it in this pull request.

Any other opinions ?

File: docs/building.md (hunk 1 above)
Comment (jman-schief) [reply docs/building.md:36]: I think this block about Docker is confusing (I got it wrong) so I'll remove it. As you suggest let's add later a section about building with Docker

File: docs/building.md (hunk 2 above)
Comment (jman-schief) [reply docs/building.md]: moved the build requirements in the new `building` section

**Comment by jman-schief** [comment]
> ```
>   - Home:
>     - Overview: README.md
>     - Setup:
>       - Build from source: building.md
>       - Release deb package: release.yml
>       - Nix Build / Develop: nix-develop.md
>     - Getting started: getting-started.md
> ```

@brotzeit I've tried reworking the index as per your suggestions, how does it look now?  

I feel the indexes is somehow slightly confusing, but I'm not sure on how to improve. Maybe makes more sense to iterate on it.

grafik


(now I'm pushing randomly to this branch, before merging I'll rebase in a single commit)

**Comment by brotzeit** [comment]
> Maybe it makes more sense to iterate on it.

Agreed. We will keep improving the docs anyway.

**Review by brotzeit — APPROVED** [review]

**Review by black7375 — APPROVED** [review]

**Pull request closed without merging**